### PR TITLE
Add blog post for upcoming cordova-android@7.1.3 patch release

### PR DESCRIPTION
**Pull request by brodybits**
for review, with changes curated from apache/cordova-android#555

### CODE REVIEW

**Comment by jcesarmobile** [comment]
Is this WIP or you finished?
If you finished, there are a lot of changes missing

Incorrect default sdk version issue fix (PR #495) & update comments in build.gradle (PR #496)
Remove obsolete check for JellyBean to work properly on Android Pie (GH-534) (PR #544)
GH-539 & GH-540 fix destination path fallback & fix for source-file with app target-dir, with unit test updates (PR #542) (540 is on the list, but 539 isn't)
add missing cast for cdvMinSdkVersion (GH-551)
Multiple build-extras.gradle locations (GH-552)

**Comment by brodybits** [comment]
My bad, just marked as WIP until I fix it!

**Comment by brodybits** [comment]
I fixed the blog post, would appreciate a quick review (expecting to publish patch release and blog post later today).

**Comment by janpio** [comment]
Some of the changelog items link to PRs, some to items. 

When JIRA was a thing, they all linked to the issue they fixed. Now we don't always have issues, and it is not really defined how to handle this. In my opinion it makes sense to link all to the PR, as the PR usually includes a link to the issue it closes. Agree? Or issue and PR both in a specific way?

**Comment by brodybits** [comment]
> In my opinion it makes sense to link all to the PR, as the PR usually includes a link to the issue it closes.

I'm fine to do it that way for now, would like to see feedback from others.

> A proper "Squash and merge" commit message for example includes the PR number in the commit message: `cordova-android 7.1.3 blog post (#911)` (for this PR). Maybe we should use this template, link the `#911` to the PR and add an issue in front if available?

- departure from convention of using CB-??? or GH-??? that I saw elsewhere, looks like this came from Apache convention
- just using `#???` is not so good since we have a toolset with quite a few different repos. I would favor something like `apache/cordova-android#550` to refer to PR `#550` on `cordova-android`, like how GItHub references issues & PRs.
- moving the reference from the beginning to the end is yet another departure

**Comment by brodybits** [comment]
I just made the changes requested by @janpio. I think the requested changes make sense, and just a plain `#???` reference is fine if the document refers to changes on only one project. Will merge later today if there are no further objections.

File: www/_posts/2018-11-21-cordova-android-7.1.3.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    cordova platform add android@7.1.3
+
+<!--more-->
+
+## Curated Changelog
+
+* Incorrect default sdk version issue fix ([#495](https://github.com/apache/cordova-android/pull/495))
+* Update comments in `build.gradle` ([#496](https://github.com/apache/cordova-android/pull/496))
+* Fix dest overwrite, in case of of plugin `source-file` element with `target-dir` that does not need remapping ([#539](https://github.com/apache/cordova-android/pull/539))
+* Support plugin `source-file` element with any app `target-dir` value [#542](https://github.com/apache/cordova-android/pull/542)
```
Comment (jcesarmobile) [inline www/_posts/2018-11-21-cordova-android-7.1.3.md]: this one is missing the () around the link

Comment (brodybits) [reply www/_posts/2018-11-21-cordova-android-7.1.3.md]: Should be fixed now, thanks for checking

**Comment by brodybits** [comment]
Thanks @jcesarmobile & @janpio for reviewing. I will merge this tonight when I publish the patch release.

**Comment by janpio** [comment]
Good points @brodybits. Let's keep it as you changed it for now - and move this into a later discussion about how we build our changelogs somewhere else.

**Pull request merged**